corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -145,7 +145,7 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
       </div>
     <% } %>
 
-  <div class="<%- prefix %>-cell-grid-style">
+  <div class="<%- prefix %>-cell-grid-style collapsible-tile-content">
```

Comment:
Is there a way of conditionally applying this class? To me this implies that all case tiles are "collapsible content".